monkey/infection_monkey/control.py

```diff
@@ -321,3 +323,29 @@ def get_pba_file(filename):
                                 proxies=ControlClient.proxies)
         except requests.exceptions.RequestException:
             return False
+
+    @staticmethod
+    def should_monkey_run(vulnerable_port: str) -> bool:
```

Comment:
Port is a string?

Reply:
Yes, we are not intending to do any math operations with it, just send it as data.